gebru/roster.json

```diff
@@ -81,6 +81,11 @@
     {
       "name": "Dan Willis",
       "github": "WalkingZ3d"
+    },
+    {
+      "name": "Michael Harkins-Meloy",
+      "github": "Harkins12345",
+      "myPets": [{"cat": "Dexter"}, {"chickens": ["Shirley", "Hettie"]}]
```

Comment:
I am not sure what prompted you to add those? :)

Reply:
Well technically the hello github tutorial never said what changes I should make specifically, so I decided to try and show my understanding of the JSON format. I can of course remove them if they're not the changes you were looking for

Reply:
Never mind I didn't read it properly